src/GitVersion.Core/VersionCalculation/VersionSearchStrategies/VersionIncrementSourceType.cs

```diff
@@ -0,0 +1,9 @@
+namespace GitVersion.VersionCalculation;
+
+public enum VersionIncrementSourceType
```

Comment:
Can you please explain me for what the enum `VersionIncrementSourceType` is good for? In my opinion the hole SourceType property can be removed.